regparser/federalregister.py

```diff
@@ -25,7 +31,10 @@ def fetch_notice_json(cfr_title, cfr_part, only_final=False,
         params["conditions[type][]"] = 'RULE'
     if max_effective_date:
         params["conditions[effective_date][lte]"] = max_effective_date
-    response = requests.get(API_BASE + "articles", params=params).json()
+    url = API_BASE + "articles"
+    logger.info("Fetching notices - URL: %s Params: %r", url, params)
```

Comment:
1. Did you mean this:
2016-02-23 12:10:07 requests.packages.urllib3.connectionpool Starting new HTTPS connection (1): www.gpo.gov
What exactly are you suggesting?
2.  A couple of lines below, we have: 
```
logger.debug("Fetching notices response - %r", response)
```
This will have the `from_cache` attribute appropriately set. 
I would think that if we are logging at the info level, we have a clue to whether the cache is used by looking at the timestamps. At the debug level, as mentioned, we have the `response.from_cache` attribute.